resources/prometheus/prometheus-rules.yaml

```diff
@@ -214,7 +214,7 @@ spec:
           record: rhacs_operator:namespace:workload:container:max_memory_usage_ratio
         - alert: RHACSOperatorMemoryUtilizationHigh
           expr: |
-            rhacs_operator:namespace:workload:container:max_memory_usage_ratio > 0.6
+            rhacs_operator:namespace:workload:container:max_memory_usage_ratio > 0.7
```

Comment:
Should we put this even higher? 80% maybe?

Reply:
That seems a bit too close to the critical threshold of 90%. Let's to baby steps.